home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/NewsServiceImpl.java

```diff
@@ -67,29 +72,32 @@ public NewsDto update(Long id, NewsDto newsDto) {
             return mapper.convert(fromDB, NewsDto.class);
 
         } else {
-            throw new NotFoundHomeException("Can't find news with given ID:" + id);
+            throw new NotFoundHomeException(NOT_FOUND_NEWS + id);
```

Comment:
Let's use `String.format` instead of concatenation.

Reply:
Substituted with String.format constant.